external/buildscripts/build_runtime_vs.pl

```diff
@@ -40,6 +40,11 @@ sub CompileVCProj
 	my $config;
 
 	my $msbuild = $ENV{"ProgramFiles(x86)"}."/MSBuild/$msBuildVersion/Bin/MSBuild.exe";
+	
+	if ((! -d $extraBuildTools) || ($ENV{YAMATO_PROJECT_ID}) || ($ENV{USERNAME} == "bokken"))
```

Comment:
We want to use 15.0 on Yamato and GitLabCI Bokken runners (where username is "bokken" if its bokken runner).

Buildfarm runners have 14.0.